dev-python/rsa/files/rsa-4.8-includes.patch

```diff
@@ -0,0 +1,31 @@
+From 3031bf5c6ae64083431e849903b0104d2cfae893 Mon Sep 17 00:00:00 2001
```

Comment:
Missing a link to the patch but I'll throw it in.

Reply:
thanks!